api/src/test/java/com/debugmafia/demo/DemoApplicationTests.java

```diff
@@ -1,4 +1,4 @@
-package com.debugmafia.demo;
+package com.debugmafia.clueless;
```

Comment:
This package is correct - but this file should be moved from:

```
api/src/test/java/com/debugmafia/demo
```
to:

```
api/src/test/java/com/debugmafia/clueless
```

Directory structure should match the source code (everything under `api/src/main/java`)